packages/control-property-editor/src/App.tsx

```diff
@@ -8,7 +8,7 @@ import { PropertiesPanel, LeftPanel } from './panels';
 import { useLocalStorage } from './use-local-storage';
 import type { RootState } from './store';
 import { useAppDispatch } from './store';
-import { changePreviewScale } from './slice';
+import { changePreviewScale, setProjectScenario } from './slice';
```

Comment:
what has the setProjectScenario functionality to do with this PR?

Reply:
It updates the store with the Scenario that was passed as props to the start method. The previous functionality was removed: https://github.com/SAP/open-ux-tools/pull/1553/files/ef164f5d741d8744ebae0aab55c6ac41e8bdf4a8#diff-48a8809da86b88a63a1cc54193f7774fdb7d06ff66f1282cd7e8fd31efb90e94L57

Reply:
Ok but again what has it to do with "Remove label and icon in control property editor". I don't understand why it is required for removing label and icon.

Reply:
Ok, I get now what you are saying. It's not required true, but we do need this change. Do you think I should open a different PR just for the scenario change? Or add a comment to the PR's description of this addition?

Reply:
Done, thanks.